t/jv-formats.t

```diff
@@ -23,17 +24,22 @@ my $schema = {type => 'object', properties => {v => {type => 'string'}}};
     for ('2017-03-29T23:02:55.831Z', '2017-03-29t23:02:55.01z', '2017-03-29 23:02:55-12:00',
     '2016-02-29T23:02:55+05:00');
 
-  validate_ok {v => 'xxxx-xx-xxtxx:xx:xxz'},       $schema, E('/v', 'Does not match date-time format.');
-  validate_ok {v => '2017-03-29\t23:02:55-12:00'}, $schema, E('/v', 'Does not match date-time format.');
-  validate_ok {v => '2017-03-29T23:02:60Z'},       $schema, E('/v', 'Second out of range.');
-  validate_ok {v => '2017-03-29T23:61:55Z'},       $schema, E('/v', 'Minute out of range.');
-  validate_ok {v => '2017-03-29T24:02:55Z'},       $schema, E('/v', 'Hour out of range.');
-  validate_ok {v => '2017-03-32T23:02:55Z'},       $schema, E('/v', 'Day out of range.');
-  validate_ok {v => '2017-02-30T23:02:55Z'},       $schema, E('/v', 'Day out of range.');
-  validate_ok {v => '2017-02-29T23:02:55Z'},       $schema, E('/v', 'Day out of range.');
-  validate_ok {v => '2017-03-00T23:02:55Z'},       $schema, E('/v', 'Day out of range.');
-  validate_ok {v => '2017-13-29T23:02:55Z'},       $schema, E('/v', 'Month out of range.');
-  validate_ok {v => '2017-00-29T23:02:55Z'},       $schema, E('/v', 'Month out of range.');
+  validate_ok {v => 'xxxx-xx-xxtxx:xx:xxz'},        $schema, E('/v', 'Does not match date-time format.');
+  validate_ok {v => "2017-03-29\t23:02:55-12:00"},  $schema, E('/v', 'Does not match date-time format.');
+  validate_ok {v => '2017-03-29T23:02:55-12'},      $schema, E('/v', 'Does not match date-time format.');
+  validate_ok {v => '2017-03-29T23:02:55+0:0'},     $schema, E('/v', 'Does not match date-time format.');
+  validate_ok {v => '2017-03-29T23:02:55+123:00'},  $schema, E('/v', 'Does not match date-time format.');
+  validate_ok {v => '2017-03-29T23:02:55+45:00'},   $schema, E('/v', 'Time offset hour out of range.');
+  validate_ok {v => '2017-03-29T23:02:55+23:60'},   $schema, E('/v', 'Time offset minute out of range.');
+  validate_ok {v => '2017-03-29T23:02:60Z'},        $schema, E('/v', 'Second out of range.');
+  validate_ok {v => '2017-03-29T23:61:55Z'},        $schema, E('/v', 'Minute out of range.');
+  validate_ok {v => '2017-03-29T24:02:55Z'},        $schema, E('/v', 'Hour out of range.');
+  validate_ok {v => '2017-03-32T23:02:55Z'},        $schema, E('/v', 'Day out of range.');
+  validate_ok {v => '2017-02-30T23:02:55Z'},        $schema, E('/v', 'Day out of range.');
+  validate_ok {v => '2017-02-29T23:02:55Z'},        $schema, E('/v', 'Day out of range.');
+  validate_ok {v => '2017-03-00T23:02:55Z'},        $schema, E('/v', 'Day out of range.');
+  validate_ok {v => '2017-13-29T23:02:55Z'},        $schema, E('/v', 'Month out of range.');
+  validate_ok {v => '2017-00-29T23:02:55Z'},        $schema, E('/v', 'Month out of range.');
```

Comment:
Please add some "ok" tests for the time offset.

Reply:
There already are some at the begin of the tests

https://github.com/mojolicious/json-validator/blob/bf879a91c804423be29adf722a05c03f925ac8b7/t/jv-formats.t#L29

I added two more with offsets +23:59 and -23:59